libs/ledgerjs/packages/types-cryptoassets/src/index.ts

```diff
@@ -177,15 +176,10 @@ export type LedgerExplorerId =
   | "hsr"
   | "kmd"
   | "ltc"
-  | "ppc"
   | "pivx"
   | "posw"
   | "qtum"
-  | "xsn"
```

Comment:
was that voluntary?

Reply:
Yes
cf. https://github.com/LedgerHQ/ledger-live/pull/7138#issuecomment-2180117026

Reply:
Ah yes, only read xst. 👍